packages/oi4-oec-service-node/src/Utilities/Helpers/ClientPayloadHelper.ts

```diff
@@ -63,34 +63,25 @@ export class ClientPayloadHelper {
         return {abortSending: false, payload: payload};
     }
 
-    createLicenseSendResourcePayload(applicationResources: IOI4ApplicationResources, filter: string, dataSetWriterIdFilter: number, resource: string): ValidatedPayload {
+    createLicenseSendResourcePayload(applicationResources: IOI4ApplicationResources, dataSetWriterIdFilter: number, resource: string, filter: string = undefined): ValidatedPayload {
```

Comment:
According to the spec, we need to "filters" for the license; <oi4Identifier> and <licenseId>.
It is possible to only provide the oi4Identifier, but not the licenseId. Therefore I suggest the following method signature:
```suggestion
    createLicenseSendResourcePayload(applicationResources: IOI4ApplicationResources, oi4Identifier?: string, licenseId?: string = undefined): ValidatedPayload {
```

Reply:
discussed and fixed together